docs-md/admission-controller.md

```diff
@@ -9,15 +9,23 @@ configuration through dashboard visibility, but to actually enforce it with this
 Note that Polaris will not alter your workloads, only block workloads that don't conform to the configured policies.
 
 ## Installation
+A valid TLS certificate is required for the Polaris Validating Webhook. If you have cert-manager installed in your cluster then the install methods below will work.
+
+If you don't use cert-manager, you'll need to:
+
+Supply a CA Bundle with the webhook.caBundle
+Create a TLS secret in your cluster with a valid certificate that uses that CA
+Pass the name of that secret with the webhook.secretName parameter.
```

Comment:
Can you add a `* ` in front of these to bullet them? Looks like they get squished together currently.

Reply:
Done, thanks for catching that.